core/src/main/java/co/cask/wrangler/steps/transformation/RowHash.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright © 2016-2017 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.wrangler.steps.transformation;
+
+import co.cask.wrangler.api.*;
+import org.apache.commons.codec.digest.DigestUtils;
+
+import java.util.List;
+
+@Usage(
+        directive = "rowhash",
+        usage = "rowhash <column> <codec>",
+        description = "Returns a hash value for the contents of the entire record"
```

Comment:
Returns/Computes? (this does not really return anything, it sets a new column to the hash value)